lib/Suggestions.js

```diff
@@ -110,8 +119,10 @@ class Suggestions extends Component {
         ref={(elem) => {
           this.suggestionsContainer = elem;
         }}
-        className={this.props.classNames.suggestions}>
-        <ul> {suggestions} </ul>
+        className={classNames.suggestions}>
+        <ul>
+          {suggestions.map((item, i) => this.renderSuggestion(item, i))}
```

Comment:
move the map logic inside `this.renderSuggestion` and rename `this.renderSuggestion` to `this.renderSuggestions`